syntax = "proto3";

package order;

import "user.proto";

service OrderService {
    rpc CreateOrder (Order) returns (OrderResponse);
    rpc GetOrder (OrderId) returns (Order);
    rpc UpdateOrder (Order) returns (OrderResponse);
}

message Order {
    int32 id = 1;
    string title = 2;
    string description = 3;
    int32 user_id = 4;
}

message OrderId {
    int32 id = 1;
}

message OrderResponse {
    string message = 1;
}